src/services/TaskRepository.ts

```diff
@@ -1,83 +1,85 @@
-import { QueryTypes } from "sequelize";
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
At the end we want a "in memory" repo a "raw query" repo and a "model" repo please make sure we are preserving the older implementations of the repository